src/chapters/orm/exercises.rst

```diff
@@ -0,0 +1,75 @@
+Exercises
+=========
+
+For the exercises, we are going to continue building on our ``coding-events`` application.
+Create a new branch for the exercises off of the ``persistent-controller-refactor`` branch. 
+
+The ``EventCategory`` Class
+---------------------------
+
+First, create a new class called ``EventCategory`` in the ``models`` directory.
+
+``EventCategory`` needs to have the following:
+
+#. An ``id`` field of type ``int``.
+#. A ``name`` field of type ``String``.
+#. A constructor.
+#. The appropriate getters and setters.
+
+``EventCategory`` represents data that will be stored in our database so you need to use the ``@Entity`` annotation!
+
+The ``EventCategoryRepository``
+-------------------------------
+
+Once you have created ``EventCategory``, you need to create the ``EventCategoryRepository`` in the ``data`` folder.
+``EventCategoryRepository`` will extend ``EventCategory``.
+
+The ``EventCategoryController``
+-------------------------------
+
+Create the ``EventCategoryController`` in the ``controllers`` directory.
+You can use the ``@RequestMapping`` annotation to map to ``"eventCategories"``.
+To get our handlers working, we also need a variable of type ``EventCategoryRepository``.
+
+We will be creating 3 handlers in our controller:
+
+#. ``displayAllEvents``
+#. ``renderCreateEventCategoryForm``
+#. ``processCreateEventCategoryForm``
+
+``displayAllEvents``
+^^^^^^^^^^^^^^^^^^^^
+
+``displayAllEvents`` needs to do the following:
+
+#. Use ``@GetMapping`` and return ``"eventCategories/index"``.
+#. Add an attribute for the ``title`` that uses ``"All Categories"``.
+#. Add an attribute for the ``categories`` that uses all of the values in your ``EventCategoryRepository`` variable.
```

Comment:
Added a note to the top with the relevant links for students for what these classes should look like. I want to link to the text sections, but since they are in a different branch, I linked straight to Youtube for now. I also added a section on what the final app will look like.